humanlayer-wui/src/components/internal/SessionDetail/components/CustomDiffViewer.tsx

```diff
@@ -600,19 +600,20 @@ export const CustomDiffViewer = ({
         </div>
       </div>
     )
-  } catch (error) {
+  } catch {
     // Log detailed context for debugging
-    logger.warn('Snapshot-based diff rendering failed, falling back to simple diff:', {
-      error: error instanceof Error ? error.message : String(error),
-      editsCount: edits.length,
-      fileContentLength: fileContents?.length,
-      firstEditPreview: edits[0]
-        ? {
-            oldValue: edits[0].oldValue.substring(0, 50) + '...',
-            newValue: edits[0].newValue.substring(0, 50) + '...',
-          }
-        : null,
-    })
+    // Commented out (not deleted) for future debugging if needed
+    // logger.warn('Snapshot-based diff rendering failed, falling back to simple diff:', {
+    //   error: error instanceof Error ? error.message : String(error),
+    //   editsCount: edits.length,
+    //   fileContentLength: fileContents?.length,
+    //   firstEditPreview: edits[0]
+    //     ? {
+    //         oldValue: edits[0].oldValue.substring(0, 50) + '...',
+    //         newValue: edits[0].newValue.substring(0, 50) + '...',
+    //       }
+    //     : null,
+    // })
```

Comment:
hmm don't think I like having big commented blocks in the code, how about, higher up in the file or something

```
 debugLogSnapshotFailure(...) {
   // original logic, not commented out
 }
```
and then in the function itself, just comment out the invocation

```
} catch {
     // debugLogSnapshotFailure(…)
}
```
then we can leave the import uncommented too